src/PostgREST/Types.hs

```diff
@@ -54,12 +54,12 @@ data PrimaryKey = PrimaryKey {
 } deriving (Show, Eq)
 
 data OrderDirection = OrderAsc | OrderDesc deriving (Eq)
-instance Show OrderDirection where
+instance S.Show OrderDirection where
```

Comment:
I'm assuming we'll change this to the protolude Show instance once other modules are converted to protolude?